elastic/beats
# [Endpoint] Add action type for endpoint

## What does this PR do?

routes `endpoint` action types to the `endpoint` 

## Why is it important?



## Checklist



~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~


## How to test this PR locally




Spin up fleet server, and a regular agent configured with endpoint integration. Write an action to the fleet actions index and observe the agent log to see that the message is delivered

```http
POST /.fleet-actions/_doc 

{
	"action_id": "f0d4e6bf-22f6-4898-afed-9a0aba03452f",
	"type": "INPUT_ACTION",
	"input_type": "endpoint",
	"@timestamp": "2021-04-26T14:48:21.581Z",
	"expiration": "2021-07-23T12:18:55-04:00",
	"agents": [
		"<PUT REAL AGENT ID HERE>"
	],
	"data": {
		"command": "isolate"
	}
}
```

Observe agent logs similar to:

```
2021-04-26T11:56:59.448-0400	DEBUG	dispatcher/dispatcher.go:83	Dispatch 1 actions of types: *fleetapi.ActionApp
2021-04-26T11:56:59.448-0400	DEBUG	handlers/handler_action_application.go:36	handlerAppAction: action 'action_id: 1c4cc421-32c5-494c-9868-1b2249cc8eac, type: INPUT_ACTION, input_type: endpoint' received
```
